designs/administrator/olm/custom-operator-delete-message/index.md

```diff
@@ -0,0 +1,17 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Custom message on operator uninstall
+
+When an operator is uninstalled by an admin, the operator's developer may like to show a custom message in the uninstall dialog that might warn the admin of any consequences of uninstalling it, that might be particular to that operator.
+
+For example, when a storage operator is uninstalled, perhaps any associated data in that storage is also wiped, which the operator's developer would like to convey to the user.
+
+**Note:** The custom warning text used in this design is just for example-sake, and not actually intended to be use.
+
+## Uninstalling operator
+
+![Cluster dashboard status card message click](img/1-1-header.png)
+- When the operator developer has specified a string in the new 'custom delete message' annotation, a new fixed header of **Operator developer message** appears in the uninstall operator dialog with the string beneath it
```

Comment:
I'd expect the modal to read something different than **Operator developer message**. Maybe something like:

**<Operator name> suggests the following**

Reply:
Maybe "Message from OpenShift Container Storage"

Reply:
Since we can't be sure that the developer's message will include a "suggestion", calling it a generic "message" seems safer. If the second sentence shown in this screenshot weren't there for example, the wording would be off.

"Message from OpenShift Container Storage" or maybe the more personal "Message from the operator developer" sounds good to me.

Reply:
Included this change, thanks.